### PR TITLE
EPMCUXDA-6553 (Auth integration)

### PR DESCRIPTION
**Pull request by yanaSelin**
### What does this PR do?

`Add ability to navigate through direct links`

### Jira ticket related to this PR

- [EPMCUXDA-6553](https://jira.epam.com/jira/browse/EPMCUXDA-6553)

### Link to branch and/or PR's related

`Just remove section if it isn't related to anything`

- https://github.com/abahachuk/ptushki-frontend/pull/8

#### Type of change

`What types of changes does your code introduce? Put an 'x' in all the boxes that apply:`

- [ ] Adds new features.
- [x] Changes existing functionality.
- [ ] Deprecates features.
- [ ] Removes features.
- [ ] Fixes any bug.
- [ ] Other: security, build process, infrastructure, tests, docs, etc.

### CODE REVIEW

**Review by PaulGiletich — APPROVED** [review]

File: src/services/SecurutyService.ts [hunk 1]
```diff
@@ -1,16 +1,49 @@
+/* eslint-disable class-methods-use-this */
 import { UserInfo } from "../app/features/auth/models";
 
 const REFRESH_TOKEN = "refreshToken";
 const ACCESS_TOKEN = "accessToken";
 const USER_INFO = "userInfo";
 
+export class SecurityError extends Error {
```
Comment (PaulGiletich) [inline src/services/SecurutyService.ts:8]: is there a reason for this?

File: src/services/SecurutyService.ts [hunk 2]
```diff
@@ -1,16 +1,49 @@
...
+  constructor(message: string) {
+    super();
```
Comment (PaulGiletich) [inline src/services/SecurutyService.ts:10]: anyways, i think you can call super(message) instead of setting message yourself

Comment (yanaSelin) [reply src/services/SecurutyService.ts:10]: According to [this source](https://learn.javascript.ru/oop-errors) parent Error constructor doesn't set message property.

File: src/services/SecurutyService.ts (hunk 1 above)
Comment (yanaSelin) [reply src/services/SecurutyService.ts:8]: To be able to detect security errors (like expired tokens) in actions and sign user out.

**Pull request merged**